src/algebra/algebra/basic.lean

```diff
@@ -1135,6 +1135,10 @@ end of_linear_equiv
 
 @[simp] lemma mul_apply (e₁ e₂ : A₁ ≃ₐ[R] A₁) (x : A₁) : (e₁ * e₂) x = e₁ (e₂ x) := rfl
 
+@[simp] lemma mul_eq_trans {a b : A₁ ≃ₐ[R] A₁} : a * b = b.trans a := rfl
```

Comment:
Isn't this `alg_equiv.aut_mul`?
https://leanprover-community.github.io/mathlib_docs/algebra/algebra/basic.html#alg_equiv.aut_mul

Reply:
Good point! I've removed it :)